src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -378,4 +378,53 @@ public with sharing class AFFL_AccRecordType_TEST {
         System.assertEquals(1, allAffiliationList.size());
         System.assertEquals(true, allAffiliationList[0].Primary__c);
     }
+
+    @isTest
+    public static void checkNPE() {        
+        
+        setup();
+
+        //delete an affiliation mapping
+        List<Affl_Mappings__c> mappings = [select ID from Affl_Mappings__c where Name = 'Household Account'];
```

Comment:
This is minor because the rest of the class might be written like this as well, but for the SOQL keywords we prefer them to be capitalized - like "SELECT", "FROM", "WHERE", etc

Reply:
Honestly, lines 388-391 could just be consolidated into one line: `delete [SELECT Id FROM Affl_Mappings__c WHERE Name = 'Household Account'];` (no need to check if there are rows returned)

Reply:
@robertbwatson updated the test class.